lib/trailblazer/operation/controller.rb

```diff
@@ -70,14 +81,25 @@ def operation!(operation_class, params)
       params.merge!(concept_name => request_body)
     end
 
-    res, @operation = yield # Create.run(params)
-    setup_operation_instance_variables!
-
-    [res, @operation] # DISCUSS: do we need result here? or can we just go pick op.valid?
+    if @fetching_collection
+      @operation = yield # Create.run(params)
+      setup_operation_collection_variables!
+      @operation # DISCUSS: do we need result here? or can we just go pick op.valid?
+    else
+      res, @operation = yield # Create.run(params)
+      setup_operation_instance_variables!
+      [res, @operation] # DISCUSS: do we need result here? or can we just go pick op.valid?
+    end
   end
 
   def setup_operation_instance_variables!
     @form = @operation.contract
     @model = @operation.model
   end
+  
+  def setup_operation_collection_variables!
+    @collection = @operation.collection
+    operation_collection_name = @operation.collection.model.table_name
+    instance_variable_set(:"@#{operation_collection_name}", @collection)
```

Comment:
I'm gonna put this on https://github.com/apotonick/trailblazer/blob/master/lib/trailblazer/operation/crud.rb